parsers/ERP_PGCB.py

```diff
@@ -73,17 +77,13 @@ def parse_table_body(table_body: Tag) -> list[dict]:
 
         # date and time in [0]; [1] are DD-MM-YYYY; HH:mm[:ss]
         parsed_day = datetime.strptime(row_items[0], "%d-%m-%Y").date()
-        try:
-            assert isinstance(row_items[1], str)
-            if row_items[1][0:2] == "24":
-                # The endpoint is reporting 24:00:00 as 00:00:00 of the next day, so we need to fix that.
-                row_items[1] = row_items[1].replace("24", "00", 1)
-                parsed_day += timedelta(days=1)
-            # newer data points are in HH:mm:ss format
-            parsed_time = datetime.strptime(row_items[1], "%H:%M:%S").time()
-        except ValueError:
-            # very old data points are sometimes in HH:mm format
-            parsed_time = datetime.strptime(row_items[1], "%H:%M").time()
+        assert isinstance(row_items[1], str)
+        hour = int(row_items[1][0:2])
+        # The hour is backwards looking so if it's 01:00, it's actually 00:00
+        hour = hour - 1
+        minute = int(row_items[1][3:5])
+        second = int(row_items[1][6:8])
+        parsed_time = time(hour, minute, second)
```

Comment:
This is the only part I am unsure of, the values looks more correct when doing this and it's more logical but there are a few datapoints where both 00:00 and 24:00 exists. But then again I also found two 01:00 values for one day so hard to tell which is right.